extensions/adapters/raster/src/main/java/mil/nga/giat/geowave/adapter/raster/adapter/RasterDataAdapter.java

```diff
@@ -2079,6 +2105,37 @@ public ByteArrayId getFieldIdForPosition(
 	public void init(
 			PrimaryIndex... indices ) {
 		// TODO Auto-generated method stub
+		String indexCrsCode = null;
```

Comment:
is this set of code necessary at all?  I read it as assigned a local variable  that is unused and the index is coming into fitToIndex so this currently seems unnecessary.